lua/entities/gmod_wire_cd_lock.lua

```diff
@@ -95,6 +98,7 @@ function ENT:AttachDisk(disk)
 	    self.NoCollideConst:Remove()
 	    self.NoCollideConst = nil
 	    Wire_TriggerOutput(self, "Locked", 0)
+		Wire_TriggerOutput(self, "DiskEntity", nil)
```

Comment:
Inconsistent indentation here (your new code is sensibly indented, the old code had issues).